src/components/ClusterOverview/Inventory/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Col, Icon } from 'patternfly-react';
+
+import { getNodeErrorStatuses } from './nodeStatus';
+
+const InventoryItemStatus = ({ item }) => {
+  let ok = true;
+  if (item.kind === 'Node') {
+    ok = !item.data.some(i => getNodeErrorStatuses(i).length > 0);
+  }
+  return ok ? (
+    <div className="kubevirt-inventory__item-status">
+      <Icon type="fa" name="check-circle" size="2x" className="kubevirt-inventory__item-status-icon" />
+      <span>{item.data.length}</span>
+    </div>
+  ) : (
+    <div className="kubevirt-error-icon">
```

Comment:
is there a css for this class?

Reply:
No, there is not :) I deleted it to avoid confusion in latest patch